docs/Simorgh-Overview.md

```diff
@@ -0,0 +1,75 @@
+# Simorgh Mini Guide
+
+Simorgh consists mainly of two repositories:
+
+1. Simorgh https://github.com/bbc/simorgh/ - This is the repo that holds the core react application, the express server, Routes and the React containers.
+2. Psammead https://github.com/bbc/psammead - Psammead is the Front end component library for Simorgh, this is a mono repo consisting of a number of packages focussed on the application frontend. Each package is published to NPM and available under the `@bbc/psammead-*` namespace and are installed and used inside of the Simorgh container.
+
+## What is Simorgh
+
+Simorgh is a universal/isomorphic JS Single page application, written with Node, express and React. Currently Simorgh is designed to render news articles from the BBC's new CMS; optimo. It is also used to render front pages for our WorldService sites.
+
+Being universal JavaScript a Simorgh page renders on both the server and the client. A users initial page load will be a server side render, for optimal performance and compatibility with devices not using JS. Onward journeys (Clicking on inline Links to another page rendered by Simorgh) will then render client side, using the JS bundles downloaded during the original server side render. In this case a User is now only making a network request to fetch the JSON data and the page is rendered on the users device.
+
+## A High Level User Journey
+
+### The initial page load - Server Side Render (SSR)
+
+A request to a BBC article (https://www.bbc.co.uk/news/articles/clldg965yzjo) is passed on to the Simorgh application from Mozart.
```

Comment:
What’s Mozart?

Reply:
We can add that in a later addition, doesn't need to block this from being merged initially.

Reply:
We can add that in a later addition, doesn't need to block this from being merged initially.